_data/wai-evaluation-tools-list/submissions/94f91690-fe6c-11ef-bf56-01d18ed047fb.json

```diff
@@ -0,0 +1,58 @@
+{
+  "a11yloc": "",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results"
+  ],
+  "automated": [
+    "Component",
+    "Single page/screen",
+    "Sample of pages/screens",
+    "Entire website / software / product"
+  ],
+  "browsers": [
+    "Chrome"
+  ],
+  "comments": "",
+  "contact": "e.longhi@dunp.it",
+  "desktop": [
+    "Windows",
+    "MacOS",
+    "Linux"
+  ],
+  "features": "AccessiWeb Color Contrast Tester is a Google Chrome extension designed to evaluate the accessibility of web platforms. With this tool, you can quickly check the contrast ratio between two colors (text and background) directly on a web page, ensuring that it meets the WCAG 2.1 AA and AAA guidelines",
+  "form_name": "submission",
+  "form_version": "1",
+  "guideline": [
+    "WCAG 2.1"
+  ],
+  "language": [
+    "it"
+  ],
+  "license": [
+    "Free",
+    ""
+  ],
+  "product": [
+    "Website"
+  ],
+  "provider": "dunp scpl",
+  "publish-permission": "on",
+  "purpose": [
+    "Automated testing"
+  ],
+  "readterms": "on",
+  "release": "2025-03-10",
+  "repository": "wai-evaluation-tools-list",
+  "submission_date": "2025-03-11T11:32:58.105Z",
+  "submission_ref": "94f91690-fe6c-11ef-bf56-01d18ed047fb",
+  "technology": [
+    "HTML / XHTML",
+    "CSS"
+  ],
+  "title": "AccessiWeb Color Contrast Chrome extension",
```

Comment:
@josehilera The title of the tool is "AccessiWeb Color Contrast Tester", as described on the Chrome store and features field. Would you ask the vendor for clarification?